### PR TITLE
DTCHKINT-424 Design Implement Fallback Android

### PR DESCRIPTION
**Pull request by rondwilliams**
- Added listeners for on_fallback message coming from native Android SDK https://engineering.paypalcorp.com/jira/browse/DTCHKINT-424

### CODE REVIEW

File: src/payment-flows/native.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ const POST_MESSAGE = {
     ON_APPROVE:         'onApprove',
     ON_CANCEL:          'onCancel',
     ON_COMPLETE:        'onComplete',
-    ON_ERROR:           'onError'
+    ON_ERROR:           'onError',
+    ON_FALLBACK:        'onFallback'
```
Comment (mnicpt) [inline src/payment-flows/native.js:32]: Looks like you need to remove this one as it's not being used.

**Comment by bluepnume** [comment]
This is looking great to me! Thanks Rondale!

File: src/payment-flows/native.js (hunk 1 above)
Comment (rondwilliams) [reply src/payment-flows/native.js]: Done

**Comment by rondwilliams** [comment]
@bluepnume @mnicpt Tested and it works!

File: src/payment-flows/native.js [hunk 2]
```diff
@@ -598,6 +599,15 @@ function initNative({ props, components, config, payment, serviceData } : InitOp
...
 
+    const onFallbackCallback = () => {
+        getLogger().info(`native_message_onfallback`)
+            .track({
+                [FPTI_KEY.TRANSITION]: FPTI_TRANSITION.NATIVE_ON_FALLBACK
+            }).flush();
+        fallbackToWebCheckout();
+    };
+
+    // This is where we connect tp Firebase
```
Comment (mnicpt) [inline src/payment-flows/native.js:610]: `connect to`

Comment (mnicpt) [reply src/payment-flows/native.js:610]: I would also comment this as the place where we initialize and configure the Firebase connection.  We connect in socket.js.

**Pull request merged**